sygnal/apnspushkin.py

```diff
@@ -38,9 +38,18 @@
     TemporaryNotificationDispatchException,
 )
 from sygnal.helper.proxy.proxy_asyncio import ProxyingEventLoopWrapper
-from sygnal.notifications import ConcurrencyLimitedPushkin
+from sygnal.notifications import (
+    ConcurrencyLimitedPushkin,
+    Device,
+    Notification,
+    NotificationContext,
+)
 from sygnal.utils import NotificationLoggerAdapter, twisted_sleep
 
+if TYPE_CHECKING:
+    from .sygnal import Sygnal
```

Comment:
Could we stick to absolute imports for consistency? I'm aware this may have been copied from existing (inconsistent) code.

```suggestion
    from sygnal.sygnal import Sygnal
```

I'm about to put up a PR to convert all the existing relative imports into absolute imports. It might create conflicts with this PR unfortunately.